projects/spring-data-mongodb/project.yaml

```diff
@@ -0,0 +1,17 @@
+homepage: "https://github.com/spring-projects/spring-data-mongodb"
+language: jvm
+main_repo: "https://github.com/spring-projects/spring-data-mongodb.git"
+primary_contact: "pwebb@vmware.com"
+fuzzing_engines:
+  - libfuzzer
+sanitizers:
+  - address
+vendor_ccs:
+  - "wilkinsona@vmware.com"
```

Comment:
Shouldn't this person be under auto_cc, as they are a project maintainer, not a vendor?

Reply:
Yes, sorry. I also changed it for https://github.com/google/oss-fuzz/pull/8764